jobs/index.js

```diff
@@ -7,6 +7,7 @@ const deleteOrders = require('./delete_published_orders');
 const calculateEarnings = require('./calculate_community_earnings');
 const deleteCommunity = require('./communities');
 const nodeInfo = require('./node_info');
+const checkSolvers = require('./check_solvers');
```

Comment:
Hi @doitwithnotepad please test the code before push a commit, that way you'll see if it works or not

we are getting the same error
```javascript
error: Unhandled Rejection: TypeError: checkSolvers is not a function
```
this is because the function `checkSolvers()` is not well imported, please replace with:

```javascript
const { checkSolvers } = require('./check_solvers');
```